spiffworkflow-backend/src/spiffworkflow_backend/data_stores/kkv.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any
+
+from SpiffWorkflow.bpmn.serializer.helpers.registry import BpmnConverter  # type: ignore
+from SpiffWorkflow.bpmn.specs.data_spec import BpmnDataStoreSpecification  # type: ignore
+from SpiffWorkflow.task import Task as SpiffTask  # type: ignore
+from spiffworkflow_backend.models.db import db
+from spiffworkflow_backend.models.kkv_data_store import KKVDataStoreModel
+
+
+class KKVDataStore(BpmnDataStoreSpecification):  # type: ignore
+    """KKVDataStore."""
+
+    def _get_model(self, top_level_key: str, secondary_key: str) -> KKVDataStoreModel | None:
```

Comment:
am i reading correctly that the database columns for k1 and k2 are allowing null but these keys are needing to be strings? is there an issue here?

Reply:
Good catch, updated.